src/Api.ts

```diff
@@ -0,0 +1,53 @@
+import Predicates from './Predicates';
+import { RequestHandler, RequestCallback } from './request';
+import { Document } from './documents';
+import { ApiCache } from './cache';
+import ResolvedApi, { ApiData } from './ResolvedApi';
+import HttpClient from './HttpClient';
+
+export interface ApiOptions {
+  accessToken?: string;
+  requestHandler?: RequestHandler;
+  req?: any;
+  apiCache?: ApiCache;
+  apiDataTTL?: number;
+  proxyAgent?: any;
+}
+
+export default class Api {
+  url: string;
+  options: ApiOptions;
+  apiDataTTL: number;
+  httpClient: HttpClient;
```

Comment:
idem, on utilise les options d'api pour créer une instance et on utilise pas celle ci.

Reply:
It's used on line 44 in `get` function